modules/test/playwright/tests/object-web/listTypeDefinitions.spec.ts

```diff
@@ -89,4 +89,71 @@ test.describe('manage picklists inside the picklists portlet', () => {
 			page.getByRole('link', {name: listTypeDefinitionName})
 		).toBeVisible();
 	});
+
+	test('picklist entry must support capital letter keys', async ({
+		apiHelpers,
+		listTypeDefinitionPage,
+		page,
+	}) => {
+		await listTypeDefinitionPage.goto();
+
+		const listTypeDefinitionEntryName = 'ListTypeDefinitionEntryName';
+
+		const listTypeDefinitionEntryKey = 'ListTypeDefinitionEntryKey';
+
+		const listTypeDefinitionName = 'picklist' + getRandomInt();
+
+		await listTypeDefinitionPage.createPicklist(listTypeDefinitionName);
+
+		await listTypeDefinitionPage.addPicklistItem(
+			listTypeDefinitionName,
+			listTypeDefinitionEntryName,
+			listTypeDefinitionEntryKey
+		);
+
+		createdListTypeDefinitionName = listTypeDefinitionName;
+
+		const [response] =
+			await apiHelpers.listTypeAdmin.getFilteredListTypeDefinition(
+				'name',
+				listTypeDefinitionName
+			);
+
+		const [responseEntries]: PickListItem[] = response.listTypeEntries;
+
+		await page.waitForSelector('iframe');
```

Comment:
OK, I have created a function to load for the iframe to load. It should be added as a util function in a shared file.

```
/**
 * This function is a workaround to wait for the iframe to load.
 * If you don't pass loadState, it will wait for the 'load' event by default.
 * If you don't pass iframeCSSSelector, it selects 'iframe' by default.
*/
async function waitForIframeToLoad(page: Page, iframeCSSSelector?: string, loadState?: 'load' | 'domcontentloaded' | 'networkidle') {
	/**
	 * $ is deprecated, but a method to convert from Locator to FrameLocator
	 * is only available in Playwright 1.43, which we don't have yet.
	 * See here: https://playwright.dev/docs/api/class-locator#locator-content-frame
	*/

	const frameElement = await page.$(iframeCSSSelector ?? 'iframe');
	const frame = await frameElement.contentFrame();

	await frame.waitForLoadState(loadState ?? 'load');
}
```

Reply:
Then you replace `await page.waitForSelector('iframe');` with `await waitForIframeToLoad(page);`

Reply:
Thank you my friend!